src/lib.rs

```diff
@@ -323,7 +323,7 @@ mod contract {
         let args: NEP141FtOnTransferArgs = parse_json(sdk::read_input().as_slice())
             .sdk_unwrap()
             .try_into()
-            .map_err(|err| format!("ERR_JSON_{:?}", err))
+            .map_err(|err| err)
```

Comment:
This is unnecessary as you are doing nothing to `err`.

Reply:
I just generalized that part of the code. Also what you mean "doing nothing to err"?  The next line unwrapped it.

Reply:
I mean, the `map_err` is unnecessary.

Reply:
@joshuajbouw means this closure is the identity function, so it seems like it should not be needed. What happens if you simply remove this line?

Reply:
@birchmd Will be the same. You just are returning the same value in the `.map`. Removing `.map_err` yields the same result in this regard.

Reply:
I know. I was trying to rephrase what you said for @mrLSD 's benefit.